tests/e2e/.eslintrc.js

```diff
@@ -22,6 +22,7 @@ module.exports = {
     'object-shorthand': 'error',
     'space-before-function-paren': ['error', 'never'],
     'keyword-spacing': 'error',
+    'prefer-arrow-callback': 'error',
```

Comment:
During standup you mentioned this would be a warning. Did you mean to make this an error? I'm good with keeping it as an error.

Reply:
@chipit24 I think I said the wrong thing in standup. Are you ok with this being an error? IMO it makes it more likely that this rule will definitely be observed.

Reply:
Yup, I prefer error.